### PR TITLE
metrics scraping: Increase default to 30s

### PR DESCRIPTION
**Pull request by antiguru**
Signed-off-by: Moritz Hoffmann <mh@materialize.com>

### CODE REVIEW

**Comment by antiguru** [comment]
cc @ruchirK because we might not actually need this change!

**Review by antifuchs — APPROVED** [review]
10s seems good to me. Sorry for not catching the `_`/`-` in review!

File: doc/user/content/cli/_index.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Flag | Default | Modifies
 [`--disable-telemetry`](#telemetry) | N/A | Disables telemetry reporting.
 [`--experimental`](#experimental-mode) | Disabled | *Dangerous.* Enable experimental features.
 [`--introspection-frequency`](#introspection-sources) | 1s | The frequency at which to update [introspection sources](#introspection-sources).
-[`--metrics_scraping-frequency`](#prometheus-metrics) | 1s | The frequency at which to update [prometheus metrics](#prometheus-metrics).
+[`--metrics-scraping-frequency`](#prometheus-metrics) | 10s | The frequency at which to update [prometheus metrics](#prometheus-metrics).
```
Comment (quodlibetor) [inline doc/user/content/cli/_index.md:20]: I know this is pre-existing, but this description isn't quite right, it's more like:

> How often the `mz_metrics` table scrapes and caches [prometheus metrics](#prometheus-metrics).

**Review by ruchirK — APPROVED** [review]
LGTM!

cc: @danhhz this will reduce the volume of data in mz_metrics by a factor of 10 iiuc

**Comment by ruchirK** [comment]
btw, I don't know if anyone actually uses `mz_metrics` at the moment. maybe we should double check with product / sales that downsampling by a factor of 10 won't break anything for anyone?

**Comment by uce** [comment]
> btw, I don't know if anyone actually uses mz_metrics at the moment. maybe we should double check with product / sales that downsampling by a factor of 10 won't break anything for anyone?

Should we also add it to the release notes or do we consider this to be internals?

**Comment by antifuchs** [comment]
> Should we also add it to the release notes or do we consider this to be internals?

I think putting a note in the changelog is probably sufficient.

**Comment by danhhz** [comment]
Are we still intending to land this? No pressure, I'm just trying to figure out when persist has to deal with its write amp (this might more or less let us ignore the problem for another month)

**Comment by antiguru** [comment]
From my perspective this change is much less urgent now that #7899 has landed. In my tests, the metrics scraping is not an issue at the moment regarding loss-of-interactivity. But, I'm open to proceed with this change if there's a consensus that it's what we want in the long run.

**Comment by danhhz** [comment]
Gotcha! That answers my question (plan on it not landing but be pleasantly surprised if it does), thanks!

**Review by benesch — APPROVED** [review]

File: doc/user/content/release-notes.md [hunk 2]
```diff
@@ -46,6 +46,11 @@ Use relative links (/path/to/doc), not absolute links
 Wrap your release notes at the 80 character mark.
 {{< /comment >}}
 
+{{% version-header v0.9.2 %}}
+
+- Rename the metrics scraping interval parameter to
+  --metrics-scraping-interval (was --metrics-scraping-frequency).
+
```
Comment (benesch) [inline doc/user/content/release-notes.md:53]: I don't think the flag made it into v0.9.1 so I think we're safe to just add a release note about its addition?

File: doc/user/content/cli/_index.md (hunk 1 above)
Comment (antiguru) [reply doc/user/content/cli/_index.md]: Changed the wording.

File: doc/user/content/release-notes.md (hunk 2 above)
Comment (antiguru) [reply doc/user/content/release-notes.md:55]: Thanks, PR updates.

**Review by antifuchs — APPROVED** [review]
This looks great. Pre 1.0 it would be great if we could rename `--introspection-frequency` as well, for full naming/unit consistency (:

**Pull request merged**